src/views/Create.vue

```diff
@@ -153,60 +156,76 @@ watch(selectedNetworkId, () => {
-        </div>
-      </fieldset>
-
-      <div class="s-box">
-        <h3 class="mb-2">Settings</h3>
-        <SIObject
-          v-model="settingsForm"
-          :error="settingsFormErrors"
-          :definition="settingsDefinition"
+      <div class="mt-8 lg:mt-0">
+        <BlockSpaceFormProfile
+          v-if="currentPage === 'profile'"
+          :form="metadataForm"
+          @errors="v => handleErrors('profile', v)"
+        />
+        <BlockSpaceFormNetwork v-else-if="currentPage === 'network'" v-model="selectedNetworkId" />
+        <BlockSpaceFormStrategies
+          v-else-if="currentPage === 'strategies'"
+          v-model="votingStrategies"
+          :available-strategies="selectedNetwork.constants.EDITOR_VOTING_STRATEGIES"
+          title="Voting strategies"
+          description="Lorem ipsum..."
+        />
+        <BlockSpaceFormStrategies
+          v-else-if="currentPage === 'auths'"
+          v-model="authenticators"
+          :available-strategies="selectedNetwork.constants.EDITOR_AUTHENTICATORS"
+          title="Authenticators"
+          description="Lorem ipsum..."
+        />
+        <BlockSpaceFormStrategies
+          v-else-if="currentPage === 'executions'"
+          v-model="executionStrategies"
+          :available-strategies="selectedNetwork.constants.EDITOR_EXECUTION_STRATEGIES"
+          title="Execution strategies"
+          description="Lorem ipsum..."
+        />
+        <BlockSpaceFormVoting
+          v-if="currentPage === 'voting'"
+          :form="settingsForm"
+          @errors="v => handleErrors('voting', v)"
+        />
+        <BlockSpaceFormController
+          v-if="currentPage === 'controller'"
+          v-model="controller"
+          @errors="v => handleErrors('controller', v)"
         />
       </div>
 
-      <BlockStrategiesConfig
-        v-model="authenticators"
-        :available-strategies="selectedNetwork.constants.EDITOR_AUTHENTICATORS"
-        title="Authenticators"
-        description="Lorem ipsum..."
-        class="mt-6"
-      />
-
-      <BlockStrategiesConfig
-        v-model="votingStrategies"
-        :available-strategies="selectedNetwork.constants.EDITOR_VOTING_STRATEGIES"
-        title="Voting strategies"
-        description="Lorem ipsum..."
-        class="mt-6"
-      />
-
-      <BlockStrategiesConfig
-        v-model="executionStrategies"
-        :available-strategies="selectedNetwork.constants.EDITOR_EXECUTION_STRATEGIES"
-        title="Execution strategies"
-        description="Lorem ipsum..."
-        class="mt-6"
-      />
-
-      <UiButton class="w-full" :loading="sending" :disabled="disabled" @click="handleSubmit">
+      <UiButton
+        v-if="showCreate"
+        class="w-full"
+        :loading="sending"
+        :disabled="submitDisabled"
+        @click="handleSubmit"
+      >
         Create
       </UiButton>
+      <UiButton v-else class="w-full" :disabled="nextDisabled" @click="handleNextClick">
```

Comment:
For consistency and readability we need every prop on new line here

Reply:
It's autoformatted by Prettier.